student-store-ui/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import "./Footer.css";
+import playStore from "../assets/play-store.svg";
+import appleStore from "../assets/app-store.svg";
+
+export default function Footer() {
+  return (
+    <footer id="footer">
+      <div class="links">
+        <div class="link-column">
+          <h4>Categories</h4>
+          <ul className="link-content">
+            <li>All Categories</li>
+            <li>Clothing</li>
+            <li>Food</li>
+            <li>Accessories</li>
+            <li>Tech</li>
+          </ul>
```

Comment:
Could potentially reduce size and repetitiveness of this file by creating an object of arrays that contain the text for this items, then map through them. Check the exemplar code for this approach